### PR TITLE
Add platform guide for FRR

### PR DESCRIPTION
**Pull request by NilashishC**
Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

##### SUMMARY
- Add platform guide for FRR.
- Fixed a line in VyOS platform guide.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
user_guide/platform_frr.rst
user_guide/platform_vyos.rst

### CODE REVIEW

File: docs/docsite/rst/network/user_guide/platform_frr.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
+- If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
+
+Example CLI Task
+----------------
+
+.. code-block:: yaml
+
+   - name: Gather FRR facts
+     frr_facts:
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_frr.rst:63]: should this use FQCN?

Comment (NilashishC) [reply docs/docsite/rst/network/user_guide/platform_frr.rst:63]: Not sure about this. This guide would be valid for the FRR modules in ansible/ansible 2.9 as well and none of the other platform guides have the examples in fqcn format, afair.

Comment (samccann) [reply docs/docsite/rst/network/user_guide/platform_frr.rst:63]: Hmm perhaps I misunderstood. I thought the purpose of column Q in John's internal spreadsheet (update platform guides) was specifically to have them use collections. 

Also, what you are changing here is the 2.10 FRR platform guide. You would need to backport this to 2.9 to have it applicable/available for 2.9.  

But this brings up a bigger issue. All of the 2.9 platform guides are... for 2.9 :-)  None of them have details about collections, yet  the 1.0.0 collections will all point to these guides. 

So for now, just keep it as you have it (no FQCN) and backport please.
I'll discuss more openly what we should do to cover the fact that the batch of 1.0.0 network collections will work with 2.9 and what if anything we should put in 2.9 platform docs.

Comment (NilashishC) [reply docs/docsite/rst/network/user_guide/platform_frr.rst:63]: +1

**Pull request merged**